src/test/util/txmempool.cpp

```diff
@@ -141,6 +141,54 @@ std::optional<std::string> CheckPackageMempoolAcceptResult(const Package& txns,
     return std::nullopt;
 }
 
+std::vector<uint32_t> GetDustIndexes(const CTransactionRef tx_ref, CFeeRate dust_relay_rate)
```

Comment:
Did you mean `const CTransactionRef& tx_ref` here?

Reply:
sure, changed